appinfo/app.php

```diff
@@ -4,15 +4,21 @@
 
 use OCP\Util;
 
-Util::addStyle('dicomviewer', 'viewer');
-Util::addStyle('dicomviewer', 'sidebar');
-Util::addStyle('dicomviewer', 'bootstrap_accordion');
-Util::addStyle('dicomviewer', 'viewerMain');
-Util::addStyle('dicomviewer', 'viewerMainDialog');
-Util::addStyle('dicomviewer', 'captureImageDialog');
-Util::addStyle('dicomviewer', 'annotationDialogs');
```

Comment:
You have missed 'annotationDialogs'. Actually, I prefer a new PR for those changes in 'app.php' file since this is not a minor refactoring. Could you please revert those changes back in this PR and create another PR with only those changes (in case you don't know how to do that, you should create a new branch in your repo from development branch in this repo, then commit only those changes to your new branch, and create a new PR to merge that branch into development branch in this repo)?

Reply:
OK. I will do so. And thanks for the help. I allready tried to figure out, why my PRs came all together. Newbie! :-)